### PR TITLE
Update CONTRIBUTING.md (Take two)

### PR DESCRIPTION
**Pull request by MrsTonedOne**
@duncathan said to make this pr

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -193,6 +193,14 @@ This prevents nesting levels from getting deeper then they need to be.
 
 * Do not divide when you can easily convert it to a multiplication. (ie `4/2` should be done as `4*0.5`)
 
+####Enforced not enforced
+The following different coding styles are not only not enforced, but it is generally frown upon to change them over from one to the other for little reason:
+
+* English/British spelling on var/proc names
+	* Color/Colour nobody cares,
+* Spaces after control statements
+	* if() if () nobody cares.
```
Comment (ChangelingRain) [inline CONTRIBUTING.md]: excuse me if() is the lord

Comment (JJRcop) [inline CONTRIBUTING.md]: what if (it's not) this one is

**Comment by duncathan** [comment]
:+1:

**Comment by Incoming5643** [comment]
you might want to use a different example than color/colour since color is a reserved var in byond now.

**Comment by MrStonedOne** [comment]
>now

Its always been a reserved word, that doesn't stop you from doing var/hair_color or var/hair_colour

**Comment by Incoming5643** [comment]
Not always, but I guess that was a good long while ago.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -193,6 +193,14 @@ This prevents nesting levels from getting deeper then they need to be.
 
 * Do not divide when you can easily convert it to a multiplication. (ie `4/2` should be done as `4*0.5`)
 
+####Enforced not enforced
+The following different coding styles are not only not enforced, but it is generally frown upon to change them over from one to the other for little reason:
```
Comment (optimumtact) [inline CONTRIBUTING.md]: frowned

**Comment by Iamgoofball** [comment]
i refuse to use british spelling on shit

:us:

**Comment by Cheridan** [comment]
ok i'll mourge ye PR now before i go to the shoppe

**Pull request merged**